doc/ref_model/chapters/chapter09.md

```diff
@@ -306,3 +318,137 @@ Topics include:
+
+<a name="9.6.2"></a>
+### 9.6.2. The Architecture
+
+In geographically dispersed large cloud deployments, a given telco cloud may have several cloud infrastructure components as well a large set of virtualized workloads (VNF/CNFs). It is important to monitor all of these workloads and infrastructure components. Furthermore, it is even more important to be able to correlate between the metrics provided by these entities to determine the performance and/or issues in such deployments. 
+
+The cloud deployment tends to shrink and expand based upon the customer demand. Therefore, an architecture is required that can scale on demand and does not force a strong tie between various entities. This means, the workloads and cloud infrastructure components that provide telemetry/performance metrics must not be burdened to discover each other. The capacity (e.g. speed, storage) of one component must not force overrun or underrun situations that would cause critical data to be lost or delayed to a point to render it useless. 
+
+Operators in charge of the Cloud Infrastructure (physical infra plus virtualization platform) require very detailed alarms and metrics to efficiently run their platform. While they need indicators about how well or poorly individual virtual machines and containers run, they don’t need a view inside these workloads. In fact, what and how workloads do is none of the NFVI operators business. The architecture must allow for different consumers to grant or deny access to available resources.
+
+Multiple workloads or network services can be deployed onto one or more sites. These workloads require logical separation so that their metrics don’t mix by accident or simply based on security and privacy requirements. This is achieved by deploying these workloads within their own tenant space. All virtualization platforms offer such isolation down to virtual networks per tenant.
+
+<a name="9.6.2.1"></a>
+#### 9.6.2.1. Push Vs. Pull
+
+Two widely deployed models for providing telemetry data are pull and push. 
+
+ <a name="9.6.2.1.1"></a>
+##### 9.6.2.1.1. Pull Model
+
+Typical characteristics of a pull model are:
+
+* The consumers are required to discover the producers of the data
+* Once the producers are identified, there should be a tight relationship (synchronization) between the producer and consumer. For example, if a producer encounters a LCM (Life Cycle Management) event - such as it moves to a different location or reboots/restarts, the consumer must re-discover it and bind the relationship again - makes the systems very complex in terms of configuration as well management. 
+* Data is pulled explicitly by the consumer. The consumer must have appropriate bandwidth, compute power, and storage to deal with this data - example SNMP pull/walks
+
```

Comment:
Another problem with Pull is that both consumers and producers have to have means for load/performance regulation in cases where the set of consumers overload the pull request serving capabilities of the producer.

Reply:
Good suggestion. Added to the text